src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -69,6 +70,31 @@ CaseDetails submitForCaseworker(
             @RequestParam Map<String, Object> searchCriteria
     );
 
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/caseworkers/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/cases/pagination_metadata",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
+    PaginatedSearchMetadata getPaginationInfoForSearchForCaseworkers(@RequestHeader(AUTHORIZATION) String authorisation,
+                                                              @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorisation,
+                                                              @PathVariable String userId,
+                                                              @PathVariable String jurisdictionId,
+                                                              @PathVariable String caseType,
+                                                              @RequestParam Map<String, String> searchCriteria
+    );
+
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/citizens/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/cases/pagination_metadata",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
+    PaginatedSearchMetadata getPaginationInfoForSearchForCitizens(@RequestHeader(AUTHORIZATION) String authorisation,
```

Comment:
could you move the RequestHeader bit down to the next line please and then reformat?
it will make it easier to read the parameters

same on the block above